src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,28 @@
+@import "../utils/variables";
```

Comment:
you can avoid importing it every time, just put every import to your `main.scss`